base/src/test/java/org/aya/test/LibraryTest.java

```diff
@@ -32,15 +34,23 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class LibraryTest {
-  @Test public void testOnDisk() throws IOException {
-    FileUtil.deleteRecursively(DIR.resolve("build"));
+  @ParameterizedTest
+  @ValueSource(strings = {"success"})
+  public void testOnDisk(@NotNull String libName) throws IOException {
+    var libRoot = TestRunner.DEFAULT_TEST_DIR.resolve(libName);
+
+    FileUtil.deleteRecursively(libRoot.resolve("build"));
     // Full rebuild
-    assertEquals(0, compile());
+    assertEquals(0, compile(libRoot));
     // The second time should load the cache of 'common'.
-    FileUtil.deleteRecursively(DIR.resolve("build").resolve("out"));
-    assertEquals(0, compile());
+    FileUtil.deleteRecursively(libRoot.resolve("build").resolve("out"));
+    assertEquals(0, compile(libRoot));
     // The third time should do nothing.
-    assertEquals(0, compile());
+    assertEquals(0, compile(libRoot));
+  }
+
+  @Test public void fastTestOnDisk() throws IOException {
```

Comment:
manual test only :(

Reply:
What do you mean? CI says it already passes tes